homeassistant/components/climate/__init__.py

```diff
@@ -187,13 +150,7 @@ class ClimateDevice(Entity):
     @property
     def state(self):
         """Return the current state."""
-        if self.is_on is False:
-            return STATE_OFF
-        if self.current_operation:
-            return self.current_operation
-        if self.is_on:
-            return STATE_ON
-        return None
+        return self.hvac_state
```

Comment:
Should this be

```suggestion
        return self.operation_mode
```

?

Reply:
Is fixed but I forget to push it :D